primitives/src/policy.rs

```diff
@@ -31,6 +31,23 @@ pub const VERSION: u16 = 1;
 /// Number of available validator slots. Note that a single validator may own several validator slots.
 pub const SLOTS: u16 = 512;
 
+// The max number of request to be processed per peerID and per request type.
+
+/// The range to restrict the responses to the requests on the consensus network.
+pub const MAX_RESPONSE_REQUEST_BLOCK_RANGE: u32 = 1000;
```

Comment:
```suggestion
pub const MAX_RESPONSE_REQUEST_BLOCK_RANGE: u32 = 10;
```
if we end up just merging this prior to deciding which are the limits for the request types, I would suggest having a smaller range, so that no unwanted delays are introduced.